### PR TITLE
feat(dropdown): AR-3072 adding button dropdown component

### PR DESCRIPTION
**Pull request by aziz-zen-ab**
This pull request adding button dropdown component

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.2.5]
```
Comment (rayhan-ab) [inline CHANGELOG.md:7]: will it be used immediately? if no, then not bumping version should be fine

Comment (aziz-zen-ab) [reply CHANGELOG.md:7]: Yes it will be use immediately, there is a ticket to implements this fix.

**Review by rayhan-ab — APPROVED** [review]

**Comment by rayhan-ab** [comment]
before merging, if we want to use ui-lib Button as trigger, why don't just set it such as <DropdownMenu trigger={<Button {...props} />}? @aziz-zen-ab

**Comment by aziz-zen-ab** [comment]
@rayhan-ab the default type is not the `button` it should be the `default` since the most implementation is in `...` action dropdown:
image

Meanwhile the `button` trigger type from our design has its own styling: https://www.figma.com/file/UIz0Jg4o7EfbbUhSicXYFk/AP-1.0---General%3A-Master-Components?node-id=14%3A1446

**Comment by rayhan-ab** [comment]
i see, okay then

**Comment by aziz-zen-ab** [comment]
Exactly. ButtonDropdown is for the triggerType = button

**Pull request merged**